File: mario.py
from cs50 import get_int

size = get_int("Height: ")
while(True):
    if(size > 0 and size < 9):
        break
    else:
        size = get_int("Height: ")

for i in range(size):
    print(" "*(size-i-1), end="")
    print("#"*(i+1), end="")
    print("  ", end = "")
    print("#"*(i+1), end="")
    print()

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int a;
    do
    {
        a = get_int("What height do you want?\n");
    }
    while(a <= 0);
    int k=0;
    for(int i=0; i<a; i++) //i++ goes through the loop once then only add i by 1 so at first it is 0
    {
        k = a-1;
        for(int j=1; j <= a-i-1; j++)
            {
                printf(" ");
            }
        //printf("%i", i);
        for(int j=a; j >= a-i; j--)
            {
                printf("#");
            }
        printf("  ");
        for(int x=a; x >= a-i; x--)
            {
                printf("#");
            }
        printf("\n");
    }
    return(0);
}
